### ISSUE TITLE
(Windows) Record audio samples alongside video frames

### ISSUE THREAD

**Issue by clearlysid** [CapSoftware/scap#9]
Unlike macOS's ScreenCaptureKit, Windows doesn't capture any audio by default so we need to build that functionality too. This needs to be executed in parallel to the video capture but I am not super familiar with Windows APIs.

**Comment by clearlysid** [comment]
Closing in favour of #11
